src/content/developers/docs/networking-layer/index.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Networking layer
+description: An introduction to Ethereum's networking layer.
+lang: en
+sidebar: true
+sidebarDepth: 2
+---
+
+Ethereum's networking layer is the stack of protocols that allows nodes to find each other and exchange information. After the merge there will be two pieces of client software (execution clients and consensus clients) each with their own separate networking stack. As well as communicating with other nodes on the Ethereum network, the execution and consensus clients also have to communicate with each other. This page gives an introductory explanation of the protocols that enable this communication to take place.
+
+## Prerequisites {#prerequisites}
+
+Some knowledge of Ethereum [nodes and clients](/src/content/developers/docs/nodes-and-clients/) will be helpful for understanding this page.
+
+## The Execution Layer {#execution-layer}
+
+The execution layer's networking protocols can be divided into two stacks: the first is the discovery stack which is built on top of UDP and allows a new node to find peers to connect to. The second is the [DevP2P](https://github.com/ethereum/devp2p) stack that sits on top of TCP and allows nodes to exchange information. These layers act in parallel, with the discovery layer feeding new network participants into the network, and the DevP2P layer enabling their interactions. DevP2P is itself a whole stack of protocols that are implemented by Ethereum to establish and maintain the peer-to-peer network.
```

Comment:
```suggestion
The execution layer's networking protocols is divided into two stacks:

- the discovery stack: built on top of UDP and allows a new node to find peers to connect to
- the DevP2P stack: sits on top of TCP and enables nodes to exchange information

Both stacks work in parallel. The discovery stack feeds new network participants into the network, and the DevP2P stack enables their interactions. DevP2P is itself a whole stack of protocols that Ethereum implements to establish and maintain the peer-to-peer network.
```

Reply:
> DevP2P is itself a whole stack of protocols that are implemented by Ethereum to establish and maintain the peer-to-peer network.

This is already somewhat complex. I wonder if containing this context under the DevP2P heading would make it easier for readers to grok this.

Reply:
ok, updated in next commit.